EEGPSRCmdGen/Resources/eegpsr_cat2.txt

```diff
@@ -93,14 +96,20 @@ $ppl       = people | children | elders | men | women
 ##############################################################################
 $person    = child
 $person    = $lpdesc (man | woman | person)
+$person    = (man | woman | person) {gesture}
 $person    = (man | woman | person) $rpdesc
-$lpdesc    = {gesture} | tallest | smallest | oldest | youngest | slimmest | fattest
-$rpdesc    = dressed with $color $outfit
-$rpdesc    = wearing ($apparel | $color $outfit)
+$lpdesc    = tallest | smallest | oldest | youngest | slimmest | fattest
+$rpdesc    = dressed in an $coloran $outfita
+$rpdesc    = dressed in a $colora $outfita
+$rpdesc    = wearing ($apparel | a $colora $outfita | an $coloran $outfitan | ($colora | $coloran) $outfits)
```

Comment:
I don't think there is a rule for $outfitan is there?

Reply:
References to $outfitan should be $outfita. This has been fixed.